docs/dom-testing-library/api-queries.mdx

```diff
@@ -109,6 +109,27 @@ screen.debug(screen.getByText('test'))
 screen.debug(screen.getAllByText('multi-test'))
 ```
 
+### `screen.logTestingPlaygroundURL`
+
+For debugging using [testing-playground](https://testing-playground.com), screen
+exposes this convenient method which logs an URL that can be opened in a
```

Comment:
I think that the correct grammar for this one is "a URL" :)

Reply:
Oops! Fixed it. 😅